package.json

```diff
@@ -12,28 +12,47 @@
   "version": "1.7.2",
   "license": "GPL-3.0",
   "description": "A powerful Markdown editor for researchers.",
-  "main": "./source/main.js",
+  "main": ".webpack/main",
   "scripts": {
-    "postinstall": "electron-builder install-app-deps && yarn less && yarn handlebars && yarn reveal:build && yarn wp:prod",
-    "start": "electron .",
-    "build:quick": "electron-builder --dir",
-    "release:this": "electron-builder",
-    "release:mac": "electron-builder --mac --publish \"never\"",
-    "release:win": "electron-builder --win --publish \"never\"",
-    "release:linux": "electron-builder --linux --publish \"never\"",
-    "less": "gulp less",
-    "handlebars": "gulp handlebars",
+    "postinstall": "electron-builder install-app-deps",
+    "start": "electron-forge start",
+    "package": "electron-forge package",
+    "make": "electron-forge make",
+    "release:this": "yarn package && electron-builder",
+    "release:mac": "yarn package && electron-builder --mac --publish \"never\"",
+    "release:win": "yarn package && electron-builder --win --publish \"never\" --prepackaged out/zettlr-win32-x64",
+    "release:linux": "yarn package && electron-builder --linux --publish \"never\"",
     "lang:refresh": "node scripts/refresh-language.js",
     "csl:refresh": "node scripts/update-csl.js",
     "reveal:build": "node scripts/update-reveal.js",
-    "wp:dev": "cross-env NODE_ENV=development webpack --progress --colors",
-    "wp:prod": "cross-env NODE_ENV=production webpack --progress --colors",
-    "watch:gulp": "gulp watch",
-    "watch:wp": "cross-env NODE_ENV=development webpack --progress --colors --watch",
-    "watch": "npm-run-all --parallel watch:*",
     "test": "mocha",
     "test-gui": "node scripts/test-gui.js",
-    "lint": "eslint --ext js,vue -f html -o eslint_report.htm source resources"
+    "lint": "eslint --ext js,vue,ts -f html -o eslint_report.htm source resources"
+  },
+  "config": {
+    "forge": {
```

Comment:
Just a thought: would it make sense to use the config file format and separate the Electron Forge configuration out of the `package.json` file? It might be easier to read as it is a Javascript file and not a JSON file. It might also result in in less merge conflicts. What do you think?

https://www.electronforge.io/configuration

Reply:
This would be possible indeed. I don't have a strong preference for either version. @nathanlesage you?

Reply:
Everything that's out of the `package.json` is a good config, I don't like the `electron-builder`-config in there either

Reply:
Ok, done!